### ISSUE TITLE
Questions about the retrieval evaluator

### ISSUE THREAD

**Issue by xiaoxiper** [HuskyInSalt/CRAG#1]
Hello, I have read your paper and have a few questions. Could you please provide more details on the fine-tuning process for T5-large? Specifically, which dataset did you use and how did you fine-tune T5 for regression in order to obtain the exact score?

I would greatly appreciate it if you could answer my questions. Thank you!

**Comment by HuskyInSalt** [comment]
Hi! The relevance signals for fine-tuning the evaluator can be collected from the existing datasets. For example, PopQA provides the golden subject wiki title from wikipedia for each question. We can use that to track a not 100% relevant but rather high-quality passage. We utilized that as the relevance labels for fine-tuning the retrieval evaluator. On the other hand, the negative samples were randomly sampled and we used the version provided by Self-RAG. Specifically, the original PopQA dataset consists of 14k samples, 1,399 of which were used for testing following Self-RAG, and the remaining were used for fine-tuning to avoid information leakage. Besides, the fine-tuned evaluator was transferred and also utilized on the Bio, Pub and ARC datasets during inference. 

Hope it can help you!
